lottie/src/main/java/com/airbnb/lottie/parser/AsyncCompositionLoader.java

```diff
@@ -21,7 +22,8 @@
     try {
       return LottieComposition.Factory.fromJsonSync(params[0]);
     } catch (IOException e) {
-      throw new IllegalStateException(e);
```

Comment:
here throw a exception will case brash! If  params[0] is a invalid JSON file, it will cause crash, but user can not catch the exception.

Reply:
This is a major change. I will introduce a new API with proper error handling.

Reply:
Please revert it.